### PR TITLE
Dz/xcmp/traps rebased

### PR DESCRIPTION
**Pull request by vimukthi-git**
Rebasing https://github.com/ComposableFi/composable/pull/2430 for release v2

### CODE REVIEW

**Review by Dom-Roth — COMMENTED** [review]
2 minor issues.
Still need to test functionality.

**Review by Martin-Kiss — APPROVED** [review]
Other than the issues already raised by @Dom-Roth LGTM

File: docs/docs/internal/xcmp/xcm-examples.md [hunk 1]
```diff
@@ -65,13 +55,38 @@ https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Fkarura-rpc-0.aca-api.network#/extr
...
+**Picasso**
+
+
+[Accept open channel request to Picasso from Statemine and Karura](//https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Fpicasso-rpc.composable.finance#/extrinsics/decode/0x2900010100020c000400000000070010a5d4e81300000000070010a5d4e8000600035e2e5d713c1800083c01e80300003c01d0070000)
+
+[Sending any asset to Statemine would cost 0.001 KSM](https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Fpicasso-rpc.composable.finance#/extrinsics/decode/0x02003b02e8030000010100a10f0100ca9a3b000000000000000000000000)
+
+**Local**
+
+[Sending any asset to Statemine would cost 0.001 KSM](https://polkadot.js.org/apps/?rpc=ws%3A%2F%2F127.0.0.1%3A9988#/extrinsics/decode/0x02003b02e8030000010100a10f0100ca9a3b000000000000000000000000)
```
[the remark this answers is not in the document]
Comment (vimukthi-git) [reply docs/docs/internal/xcmp/xcm-examples.md:89]: As not a critical issue, we will handle in a later PR.

File: docs/docs/internal/xcmp/xcm-examples.md [hunk 2]
```diff
@@ -65,13 +55,38 @@ https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Fkarura-rpc-0.aca-api.network#/extr
...
 
-[Accept open channel request to Picasso](https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Fpicasso-rpc.composable.finance#/extrinsics/decode/0x2900010100020c0004000000000700e876481713000100000700e876481700060102286bee183c01e8030000)
+**Testnet Dali Rococo**
+
+[Sending any asset to Statemine would cost 0.001 KSM](https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Frpc.composablefinance.ninja#/extrinsics/decode/0x02003702e8030000010100a10f0100ca9a3b000000000000000000000000)
+
+**Picasso**
+
+
+[Accept open channel request to Picasso from Statemine and Karura](//https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Fpicasso-rpc.composable.finance#/extrinsics/decode/0x2900010100020c000400000000070010a5d4e81300000000070010a5d4e8000600035e2e5d713c1800083c01e80300003c01d0070000)
```
[the remark this answers is not in the document]
Comment (vimukthi-git) [reply docs/docs/internal/xcmp/xcm-examples.md:83]: As not a critical issue, we will handle in a later PR.

**Pull request merged**